examples/purchaseTesterTypescript/app/screens/PaywallScreen.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Paywall } from 'react-native-purchases-ui';
+
+import {
+  StyleSheet,
+  View,
+} from 'react-native';
+import { NativeStackScreenProps } from "@react-navigation/native-stack";
+import RootStackParamList from "../RootStackParamList";
+
+type Props = NativeStackScreenProps<RootStackParamList, 'Paywall'>;
+
+const PaywallScreen: React.FC<Props> = () => {
+  return (
+    <View style={styles.container}>
+      <Paywall/>
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
```

Comment:
Should we expose `<Paywall />` with these instead of having to do that outside?

Reply:
I did that in https://github.com/RevenueCat/react-native-purchases/pull/799/files#diff-d13cb7cc4c8c4d19c22ebd27a4d83b5962c60891a42931f833fb6ee16b5e4dd8R38, it was merged in that branch from another PR so I left it there 🤷‍♂️